binder/start

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python3
+import sys
+import shutil
+import os
+
+argv = sys.argv[1:] + ['--config', 'binder/jupyter_notebook_config.py']
```

Comment:
Minor questions: Do we need to use the config.py here? can't we just pass things like `--ip=0.0.0.0`? the rest is mostly defaulted configuration?

Reply:
removed it completely